src/js/utils/mixins.js

```diff
@@ -30,17 +30,21 @@ export const fontSize = (size, lineHeight) => css`
 `;
 
 export const breakpointStyle = (breakpoint, content, responsive) => {
+  const px =
+    typeof breakpoint?.value === 'string' && breakpoint.value.endsWith('px')
```

Comment:
Do we need this additional check? In typescript, the theme-defined breakpoint values only accept number

Reply:
Doesn't hurt having it here to be more friendly, but just checking if it's needed.

Reply:
Existing breakpoints used by `breakpointStyle` are numbers so the existing code made them into a string with `px` on the end. The t-shirt size values used by this are already strings with `px` on the end. We're using the latter as the value for the media query for this situation